core/templates/components/skill-selector/select-skill-modal.component.ts

```diff
@@ -18,21 +18,19 @@
 import { Component } from '@angular/core';
 import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
 import { ConfirmOrCancelModal } from 'components/common-layout-directives/common-elements/confirm-or-cancel-modal.component';
-import { ShortSkillSummary } from 'domain/skill/short-skill-summary.model';
 import { SkillSummaryBackendDict } from 'domain/skill/skill-summary.model';
 import { SkillsCategorizedByTopics } from 'pages/topics-and-skills-dashboard-page/skills-list/skills-list.component';
-
```

Comment:
Why do you remove this.

Reply:
I removed those extra types since I changed all types to one type and therefore those extra imports were a kind of obsolete